<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>StartWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#StartWindow{border-image:url(:/images/start_bgi.png)};</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="BlueStartButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="BlueStartButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>设定变更</string>
    </property>
   </widget>
   <widget class="BlueStartButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameTitle">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>300</width>
      <height>108</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>五子棋</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="BlueStartButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>读取进度</string>
    </property>
   </widget>
   <widget class="BlueStartButton" name="recallButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>回忆棋局</string>
    </property>
   </widget>
   <widget class="BlueStartButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QWidget" name="userWidget" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>481</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#userWidget{background-color:rgb(170, 170, 255);}</string>
    </property>
    <widget class="QLabel" name="vsImage">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/vsIcon.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="whiteImage">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>白方头像</string>
     </property>
    </widget>
    <widget class="QLabel" name="blackImage">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>200</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>黑方头像</string>
     </property>
    </widget>
    <widget class="QLabel" name="whileLbl">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="blackLbl">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="userTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>开始游戏</string>
     </property>
    </widget>
    <widget class="Line" name="userTitleLine">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>30</y>
       <width>501</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="whiteChange">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>修改头像</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blackChange">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>修改头像</string>
     </property>
    </widget>
    <widget class="QComboBox" name="whiteUser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="blackUser">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="whiteNew">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>新建用户</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blackNew">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>347</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>新建用户</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gameStart">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>400</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开始游戏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>442</x>
       <y>7</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/exit.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="modeLbl">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>游戏模式：</string>
     </property>
     <property name="buddy">
      <cstring>mode</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="mode">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="whiteAILbl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>AI等级：</string>
     </property>
    </widget>
    <widget class="QLabel" name="blackAILbl">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>140</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>AI等级：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="whiteAILevel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="blackAILevel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="whiteChess">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="blackChess">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>vsImage</zorder>
    <zorder>whiteImage</zorder>
    <zorder>blackImage</zorder>
    <zorder>whileLbl</zorder>
    <zorder>blackLbl</zorder>
    <zorder>userTitle</zorder>
    <zorder>whiteChange</zorder>
    <zorder>blackChange</zorder>
    <zorder>whiteUser</zorder>
    <zorder>blackUser</zorder>
    <zorder>whiteNew</zorder>
    <zorder>blackNew</zorder>
    <zorder>gameStart</zorder>
    <zorder>exitButton</zorder>
    <zorder>modeLbl</zorder>
    <zorder>mode</zorder>
    <zorder>whiteAILbl</zorder>
    <zorder>blackAILbl</zorder>
    <zorder>whiteAILevel</zorder>
    <zorder>blackAILevel</zorder>
    <zorder>userTitleLine</zorder>
    <zorder>whiteChess</zorder>
    <zorder>blackChess</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BlueStartButton</class>
   <extends>QPushButton</extends>
   <header>bluebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
